etc/hyva-grid.xsd

```diff
@@ -171,6 +171,15 @@
         </xs:restriction>
     </xs:simpleType>
 
+    <xs:simpleType name="classType">
```

Comment:
Please remove in favor of the existing type `phpType`.

Reply:
I might have missed that, sorry